core/src/test/java/com/graphhopper/routing/ch/PrepareContractionHierarchiesTest.java

```diff
@@ -363,6 +360,57 @@ public void testUnpackingOrder_Fastest() {
         assertEquals(IntArrayList.from(10, 0, 1, 2, 3, 4, 5, 6), p.calcNodes());
     }
 
+    @Test
+    public void testDisconnects() {
```

Comment:
Ah this is unrelated to this PR and commited accidentally, but does not really hurt either ?!